views/api.py

```diff
@@ -115,7 +115,12 @@ def api_get_leaderboard():
         }
         for loved in top_loved_dicts
     ]
-    final_result = [top_loved, top_lover]
+    final_result = [
```

Comment:
The list here seems unnecessary. I‘d prefer we return just an object like

`
final_result = {
    'most_love_received': [],
    'most_love_sent': [],
}
`

Reply:
Done.